Cargo.toml

```diff
@@ -16,7 +16,7 @@ kafka = ["dep:rdkafka"]
 
 [dependencies]
 chrono = "0.4.31"
-rdkafka = { version = ">=0.29.0, <0.37.0", optional = true }
+rdkafka = { version = ">=0.29.0, <0.38.0", optional = true }
```

Comment:
Do you need to also enable `cmake-build`? I know we had to enable that in some of our other rust apps.